lib/lambda_ethereum_consensus/fork_choice/store.ex

```diff
@@ -50,10 +50,10 @@ defmodule LambdaEthereumConsensus.ForkChoice.Store do
   ##########################
 
   @impl GenServer
-  @spec init({BeaconState.t(), BeaconBlock.t()}) :: {:ok, Store.t()} | {:stop, any}
-  def init({anchor_state = %BeaconState{}, anchor_block = %BeaconBlock{}}) do
+  @spec init({BeaconState.t(), SignedBeaconBlock.t()}) :: {:ok, Store.t()} | {:stop, any}
+  def init({anchor_state = %BeaconState{}, signed_anchor_block = %SignedBeaconBlock{}}) do
     result =
-      case Helpers.get_forkchoice_store(anchor_state, anchor_block) do
+      case Helpers.get_forkchoice_store(anchor_state, signed_anchor_block.message) do
```

Comment:
I think we should also store the signed version in fork choice.

Reply:
What would be the benefit? BTW, the Store as defined in the spec, stores blocks, not signed blocks: https://eth2book.info/capella/annotated-spec/#store

Reply:
That's interesting. I was thinking we might want to validate block signatures in the fork choice.

Reply:
We do, we get `signed_block` in the `on_block` handler, then we do the validation and we store just the block